webpack/assets/javascripts/react_app/components/common/table/formatters/translatedCellFormatter.js

```diff
@@ -0,0 +1,4 @@
+import { sprintf, translate as __ } from '../../../../../react_app/common/I18n';
+import cellFormatter from './cellFormatter';
+
+export default value => cellFormatter(sprintf(__('%s'), value));
```

Comment:
How should this work?

Reply:
I tried to follow the guide Ron mentioned [here](https://github.com/theforeman/foreman/pull/9713#discussion_r1199751518).
And I found some examples with similar usages (for [example](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/forms/validators.js#L5 ))

Reply:
Thanks @ekohl!